var/spack/repos/builtin/packages/py-pyamg/package.py

```diff
@@ -25,5 +26,8 @@ class PyPyamg(PythonPackage):
     depends_on("py-numpy@1.7:", type=("build", "run"))
     depends_on("py-scipy@0.12:", type=("build", "run"))
     depends_on("py-setuptools@42:", type="build", when="@4.2.0:")
-    depends_on("py-setuptools-scm@5:+toml", type="build", when="@4.2.0:")
+    # Later version required due to
+    # https://github.com/pypa/setuptools_scm/issues/758
+    depends_on("py-setuptools-scm", type="build", when="@4.2.0:")
+    depends_on("py-setuptools-scm@7.1.0", type="build", when="@5.0.0:")
```

Comment:
```suggestion
    depends_on("py-setuptools-scm@5:+toml", type="build", when="@4.2.0:")
    depends_on("py-setuptools-scm@7.1:+toml", type="build", when="@5.0.0:")
```
Let's stick to the documented requirements as best as possible.

Reply:
Still waiting on this change

Reply:
Done.